tutorials/pages/send-shimmer-tokens-with-iota-sdk.md

```diff
@@ -0,0 +1,148 @@
+
+```sh
+# Mnemonics (Don't ever use them to manage real funds!)
+MNEMONIC="endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river"
+MNEMONIC_2="width scatter jaguar sponsor erosion enable cave since ancient first garden royal luggage exchange ritual exotic play wall clinic ride autumn divert spin exchange"
+# The Wallet database folder used to store account data
+WALLET_DB_PATH="./example-walletdb"
+# The Stronghold snapshot file location used to store secrets
+STRONGHOLD_SNAPSHOT_PATH="./example.stronghold"
+# The password to unlock the Stronghold snapshot file (Don't use it to protect real secrets!)
+STRONGHOLD_PASSWORD="24?drowssap"
+# The node URL to issue transactions with
+NODE_URL="https://api.testnet.shimmer.network"
+# The faucet URL to request test coins from
+FAUCET_URL="https://faucet.testnet.shimmer.network/api/enqueue"
+# The explorer URL to look up transactions, blocks, addresses and more
+EXPLORER_URL="https://explorer.shimmer.network/testnet"
+```
+
+## Create an account
+
+Now that we have all details, let's go through the example code.
+
+After including the needed dependencies, we have have the main function that loads the environment variables from the `.env` file we created earlier. It uses the information to set up [Stronghold](/stronghold.rs/welcome) to store our seed safely.
+
+```js
+import { Wallet, CoinType, initLogger, WalletOptions } from '@iota/sdk';
+
+// This example uses secrets in environment variables for simplicity which should not be done in production.
+require('dotenv').config({ path: '.env' });
+
+const walletOptions: WalletOptions = {
+  storagePath: process.env.WALLET_DB_PATH,
+  clientOptions: {
+    nodes: [process.env.NODE_URL],
+  },
+  coinType: CoinType.Shimmer,
+  secretManager: {
+    stronghold: {
+      snapshotPath: process.env.STRONGHOLD_SNAPSHOT_PATH,
+      password: process.env.STRONGHOLD_PASSWORD,
+    },
+  },
+};
+
+const wallet = new Wallet(walletOptions);
+
+// A mnemonic can be generated with `Utils.generateMnemonic()`.
+// Store the mnemonic in the Stronghold snapshot, this needs to be done only the first time.
+// The mnemonic can't be retrieved from the Stronghold file, so make a backup in a secure place!
+await wallet.storeMnemonic(process.env.MNEMONIC);
+
+// Create a new account
+const account = await wallet.createAccount({
+  alias: 'Alice',
+});
+console.log('Generated new account:', account.getMetadata().alias);
+```
+
+If everything worked correctly, you will see the message `Generated a new account` and you will find a Stronghold file and a database directory have been created to store the current state of your wallet.
+
+## Generate an address
+
+In this step, we will generate a new address to receive some testnet tokens. For this we will use the second example called `02_generate_address.rs`.
```

Comment:
```suggestion
In this step, we will generate a new address to receive some Testnet tokens. For this, we will use the second example called `02_generate_address.rs`.
```
Shouldn't it be `.js` or `.ts`? 

As well, if there examples are in a GH repo, could you import them as in the SDK docs?  https://github.com/iotaledger/iota-wiki/blob/main/shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx#L267-L269